modules/vba_documents/app/controllers/vba_documents/v1/reports_controller.rb

```diff
@@ -8,17 +8,23 @@ class ReportsController < ApplicationController
       skip_before_action(:authenticate)
       before_action :validate_params
 
-      MAX_REPORT_SIZE = 100
+      MAX_REPORT_SIZE = 1000
       ID_PARAM = 'ids'
 
       def create
-        statuses = VBADocuments::UploadSubmission.refresh_and_get_statuses!(params[ID_PARAM])
-        render json: statuses,
+        statuses = VBADocuments::UploadSubmission.where(guid: params[ID_PARAM])
+        render json: with_spoofed(statuses),
```

Comment:
help me understand the spoofed/fake status approach.. just curious

Reply:
So in the past we've added a not found status for any guids requested that we didn't have in the db.

Whats interesting is our inconsistency in how we handle expired payloads.... if you hit `/uploads/#{guid}` with an expired guid, in v0 it gives back the fake status with not found for the code, in v1 we straight up raise not found (giving a json api error not found). In reports though in this implementation and the previous, we actually return the record with a status of expired... I'll add that to the ticket for improving handling of expired payloads. 

~~ Just realized that while making sure I was correct in the implementation here ~~